ethereum/permissioning/src/main/java/tech/pegasys/pantheon/ethereum/permissioning/node/provider/LocalConfigNodePermissioningProvider.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.pantheon.ethereum.permissioning.node.provider;
+
+import tech.pegasys.pantheon.ethereum.permissioning.NodeLocalConfigPermissioningController;
+import tech.pegasys.pantheon.ethereum.permissioning.node.NodePermissioningProvider;
+import tech.pegasys.pantheon.util.enode.EnodeURL;
+
+public class LocalConfigNodePermissioningProvider implements NodePermissioningProvider {
```

Comment:
Do we need this class? Can't we make `NodeLocalConfigPermissioningController` implement `NodePermissioningProvider` and implement `isPermitted(final EnodeURL sourceEnode, final EnodeURL destinationEnode)` there?

Reply:
done